mitodl/salt-ops
# Salt envs

#### What are the relevant tickets?
#864 

#### What's this PR do?
Retries adding separate env configurations for pillar and state files

#### How should this be manually tested?
Apply an environment config to a minion and verify proper pillar and state rendering
